lib/tasks/db.js

```diff
@@ -297,6 +297,8 @@ function getSequelizeInstance () {
 function getMigrator (type, callback) {
   if (helpers.config.configFileExists() || args.url) {
     var sequelize = getSequelizeInstance();
+    var coffeeRegex = /^(?!skeleton)(.*)\.js$|\.coffee$/;
```

Comment:
Sorry for the delay i'll get this updated. @sdepold does the regex that @Americas introduced look good?